### DISCUSSION TITLE
Does this Project have support for python ?

### DISCUSSION THREAD

**Discussion by pre22** [fonoster/yaps#549] [General]
I'm a Software developer interested in Open Source Projects such as this one and I would be interested in working on python implementation of the project. Thank You

**Comment by psanders** [comment]
It would be interesting to have an SDK for Python. Are you part of the Discord community?

**Comment by pre22** [comment]
I'm not part of the discord community. I will like to write a python SDK for the project. Thank You

**Comment by pre22** [comment]
You can share me the link to the discord here or to my mail

**Comment by psanders** [comment]
For sure. Here is the link

https://discord.gg/4QWgSz4hTC

**Comment by vedesh-padal** [comment]
any update on the SDK for python?

**Reply by psanders** [reply]
Sadly, there is not much traction around this. 

But I'm starting to see more interest.

**Comment by jak97100** [comment]
Python support for this project?

**Reply by psanders** [reply]
Not much traction around this. What's your use-case?

**Reply by rsest** [reply]
any news about this topic?
In my case, all backend stream services are in twilio/Python/OpenAi
